web/src/app/admin/connectors/gitlab/page.tsx

```diff
@@ -0,0 +1,250 @@
+"use client";
```

Comment:
You have to also add the GitLab Connector to the Add Connector Page

Reply:
done